euphoria-hbase/src/main/java/cz/seznam/euphoria/hbase/HFileSink.java

```diff
@@ -390,6 +419,30 @@ void loadIncrementalHFiles(Path path) {
     }
   }
 
+  /**
+   * For the bulk load the files Hbase user (typically hbase) has to have ALL permissions to operate
+   * over HFiles (for move operation). The running user then has to have permissions to delete remaining folders.
+   *
+   * @param conf
+   * @param path
```

Comment:
This will trigger javadoc warning, can we add description of the two parameters or remove the `@param` at all?